general-superstaq/general_superstaq/superstaq_exceptions.py

```diff
@@ -62,3 +29,37 @@ def __init__(self, job_id: str, status: str) -> None:
             status: The status of the unsuccessful job.
         """
         super().__init__(f"Job {job_id} terminated with status {status}.")
+
+
+class SuperstaqServerException(SuperstaqException):
+    """An exception for non-retriable server-side errors.
+
+    This exception is called directly from the backend.
```

Comment:
can this sentence be reworded/removed? i'm not sure what it means

Reply:
removed